.travis.yml

```diff
@@ -14,4 +14,4 @@ script:
   - ./configure
     # 'coverage' executes all tests and checks code coverage against threshold
     # in Makefile.am
-  - make && make coverage-coveralls VERBOSE=1
+  - make && make check coverage-coveralls VERBOSE=1
```

Comment:
But then every test is executed twice. Can you add an explicit target for doing all the other steps you want to cover? Or exclude actual test execution from the `t` directory from `make check`?